buildSrc/src/main/kotlin/profiler.android-studio-setup.gradle.kts

```diff
@@ -50,7 +50,13 @@ val unpackAndroidStudio = tasks.register<Copy>("unpackAndroidStudio") {
             singleFile.name.endsWith(".tar.gz") -> tarTree(singleFile)
             else -> zipTree(singleFile)
         }
-    })
+    }) {
+        eachFile {
+            // Remove top folder when unzipping,
+            // that way we get rid of Android Studio.app folder that can cause issues on Mac
```

Comment:
:x: Could you please add a short description of the issues to expect here? A link to the original issue if appropriate would be good too.

Reply:
Added some more info